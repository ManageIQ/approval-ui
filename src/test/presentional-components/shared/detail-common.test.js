import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import ItemDetails from '../../../presentational-components/shared/detail-common';

describe('<ItemDetails />', () => {
  it('should render correctly', () => {
    const wrapper = mount(
      <ItemDetails
        toDisplay={ [ 'foo', 'bar' ] }
        foo="Foo prop"
        bar="Bar prop"
        baz="Should not be rendered"
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
