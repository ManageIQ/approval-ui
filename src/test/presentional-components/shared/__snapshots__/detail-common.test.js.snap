// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemDetails /> should render correctly 1`] = `
<ItemDetails
  bar="Bar prop"
  baz="Should not be rendered"
  foo="Foo prop"
  toDisplay={
    Array [
      "foo",
      "bar",
    ]
  }
>
  <div>
    <PropLine
      key="prop-foo"
      value="Foo prop"
    >
      <div
        className="card_element"
      >
        Foo prop
      </div>
    </PropLine>
    <PropLine
      key="prop-bar"
      value="Bar prop"
    >
      <div
        className="card_element"
      >
        Bar prop
      </div>
    </PropLine>
  </div>
</ItemDetails>
`;
